src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/Example1.txt

```diff
@@ -10,6 +10,8 @@
 public void foo(int i) {}
 public void foo(String s) {}
 public void foo(String s, int i) {}
+// comments between overloaded methods are allowed.
```

Comment:
Add this comment to Example 2 to keep everything in sync. It is like you are saying 1 is what is wrong, and 2 is what everything looks like fixed.

Reply:
Done. Just changed the comment location so the Example 2 does not look confusing and also changed the interface name used in the Example 2 from `Testing` to `ExampleInterface`